### PR TITLE
fixing compatibility with rabbitmq 3.5

### PR DESCRIPTION
**Pull request by ronydj**
Guys, I have debbuged and find a way to fix the compatibility with the newest rabbiqmq (3.5).

We need only add this method with different signature.

### CODE REVIEW

File: src/NLog.Targets.RabbitMQ/RabbitMQ.cs [hunk 1]
```diff
@@ -497,6 +497,11 @@ private void ShutdownAmqp(IConnection connection, ShutdownEventArgs reason)
 			}
 		}
 
+        private void ShutdownAmqp(object sender, ShutdownEventArgs e)
```
Comment (haf) [inline src/NLog.Targets.RabbitMQ/RabbitMQ.cs]: Whitespace

**Comment by haf** [comment]
Great! Let's fix the whitespace issue and merge!

**Comment by ronydj** [comment]
Do you need only it ?

**Comment by haf** [comment]
I'm sorry, I don't understand.

**Comment by ronydj** [comment]
look at my last commit. I have removed the whitespace

**Comment by haf** [comment]
Have you tested that the target works with your change?

**Comment by ronydj** [comment]
ITs working on my michine at least. All the tests were running ok, and I have one ELK + rabbitmq enviroment here, and they are talking each other without any problems.

**Comment by haf** [comment]
Great, let's merge it!

**Comment by haf** [comment]
Thank you for your contribution!

**Pull request merged**